corehq/apps/reports/standard/deployments.py

```diff
@@ -160,13 +160,20 @@ def selected_app_id(self):
         return self.request_params.get(SelectApplicationFilter.slug, None)
 
     @quickcache(['app_id'], timeout=60 * 60)
-    def get_app_name(self, app_id):
+    def _get_app_details(self, app_id):
         try:
             app = get_app(self.domain, app_id)
         except ResourceNotFound:
-            pass
-        else:
-            return app.name
+            return {}
+        return {
+            'name': app.name,
+            'build_profiles': {
+                profile.id: profile.name for profile in app.build_profiles
```

Comment:
if you are viewing by commit
this loop is fixed later in https://github.com/dimagi/commcare-hq/pull/26035/commits/063e460dda0a7442c2861f57cec72b77e611dbc2